app/account/page.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import React from "react";
```

Comment:
No need to import _all_ of React every time. Import only what you need, e.g. `FC`:

```suggestion
import type { FC } from "react";
```

Reply:
Has been fixed